tests/testthat/_snaps/deprecated.md

```diff
@@ -43,6 +43,14 @@
       cat_line(lifecycle_message("1.0.0", "foo(arg)", signaller = "deprecate_stop"))
     Output
       The `arg` argument of `foo()` was deprecated in base 1.0.0 and is now defunct.
+    Code
+      cat_line(lifecycle_message("1.0.0", I("Use of bananas")))
+    Output
+      Use of bananas was deprecated in base 1.0.0.
```

Comment:
I feel like something is still not quite right because these snapshots say `base`

It is possible this is just a side effect of how `lifecycle_message()` has an `env = caller_env(2)` default argument set, but won't ever actually be called by users

Reply:
Oh it looks like all the `base` snapshots get updated to `<NA>` if you use `env = global_env()` as the default, that seems better? `deprecate_warn()` and friends all pass on their own `env` argument, so the default here is only for convenience while testing and calling it interactively

Reply:
I wondered about that as well but I noticed that all the snapshots mention base as reference. So changing seems out of scope for this PR.